bot/commands/userinfo.js

```diff
@@ -0,0 +1,86 @@
+            .setTitle(channelLanguage.get('userInfoTitle'))
+            .setColor(0x2f3136)
+            .addField(channelLanguage.get('userInfoUsername'), '\`\`\`' + user.tag + '\`\`\`');
+        
+        if(member && member.nickname) embed.addField(channelLanguage.get('userInfoNickname'), '\`\`\`' + member.nickname + '\`\`\`');
+
+        embed
+            .addField("ID", '\`\`\`' + user.id + '\`\`\`')
+            .addField(channelLanguage.get('userInfoCreatedAt'), '<t:' + Math.floor(user.createdAt.getTime() / 1000) + ':R>', true);
+        
+        if (member) embed.addField(channelLanguage.get('userInfoJoinedAt'), '<t:' + Math.floor(member.joinedTimestamp / 1000) + ':R>', true);
+        if (member && member.roles.cache) embed.addField(channelLanguage.get('userInfoRoles'), '\`\`\`' + member.roles.cache.filter(c => c.name != '\@everyone').map(c => c.name).join(', ') + '\`\`\`');
+        
+        embed
+            .setThumbnail(user.avatarURL())
+            .setTimestamp();
+
+        message.reply({embeds: [embed]});
+    },
+    executeSlash: async (interaction, args) => {
+        const {channelLanguage} = interaction;
+        if(
+            interaction.guild
+            &&
+            !interaction.guild.me
+                .permissionsIn(interaction.channel)
+                .has(Permissions.FLAGS.EMBED_LINKS)
+        ) return interaction.reply(channelLanguage.get('botEmbed'));
+        const id = args ?? args[0]?.match(/^(?:<@)?!?(\d{17,19})>?$/)?.[1];
+
+        const user = id && await interaction.client.users.fetch(id).catch(() => null) || interaction.author;
+        const member = user === interaction.author ? interaction.member : interaction.guild?.members?.cache.get(id);
+        const embed = new MessageEmbed()
+            .setTitle(channelLanguage.get('userInfoTitle'))
+            .setColor(0x2f3136)
+            .addField(channelLanguage.get('userInfoUsername'), '\`\`\`' + user.tag + '\`\`\`');
+        
+        if(member && member.nickname) embed.addField(channelLanguage.get('userInfoNickname'), '\`\`\`' + member.nickname + '\`\`\`');
+
+        embed
+            .addField("ID", '\`\`\`' + user.id + '\`\`\`')
+            .addField(channelLanguage.get('userInfoCreatedAt'), '<t:' + Math.floor(user.createdAt.getTime() / 1000) + ':R>', true);
+        
+        if (member) embed.addField(channelLanguage.get('userInfoJoinedAt'), '<t:' + Math.floor(member.joinedTimestamp / 1000) + ':R>', true);
+        if (member && member.roles.cache) embed.addField(channelLanguage.get('userInfoRoles'), '\`\`\`' + member.roles.cache.filter(c => c.name != '\@everyone').map(c => c.name).join(', ') + '\`\`\`');
+        
+        embed
+            .setThumbnail(user.avatarURL())
+            .setTimestamp();
+
+            interaction.reply({embeds: [embed]});
+    },
+    slashOptions: [{
+        type: 'USER',
+        name: 'target',
+        nameLocalizations: utils.getStringLocales('banOptiontargetLocalisedName'),
```

Comment:
you forgot to change the string key
```suggestion
        nameLocalizations: utils.getStringLocales('userInfoOptiontargetLocalisedName'),
```